include/igraph_cycles.h

```diff
@@ -0,0 +1,27 @@
+
+#ifndef IGRAPH_CYCLES_H
+#define IGRAPH_CYCLES_H
+
+#include "igraph_decls.h"
+#include "igraph_interface.h"
+#include "igraph_types.h"
+#include "igraph_vector_list.h"
+
+__BEGIN_DECLS
+
+IGRAPH_EXPORT igraph_error_t igraph_fundamental_cycles(
+        const igraph_t *graph,
+        igraph_integer_t start_vid,
+        igraph_integer_t cutoff,
+        igraph_vector_int_list_t *result);
+
+IGRAPH_EXPORT igraph_error_t igraph_minimum_cycle_basis(
+        const igraph_t *graph,
+        igraph_integer_t cutoff,
+        igraph_bool_t complete,
```

Comment:
I was wondering whether we should accumulate these two booleans into a "flags" parameter where individual bits correspond to these booleans. The advantage would be that we could simply design the flags in a way that 0 means the "default" settings, i.e. the ones that a user is likely to use the most frequently.

Reply:
Okay then, mark it as `\experimental` and then we are free to change it later. I just don't like booleans in argument lists in general and even less if there is more than one of them :)